src/main/java/io/github/darealturtywurty/turtychemistry/common/block/treeblocks/rubbertree/RubberTreeBlockStripped.java

```diff
@@ -0,0 +1,52 @@
+package io.github.darealturtywurty.turtychemistry.common.block.treeblocks.rubbertree;
+
+import io.github.darealturtywurty.turtychemistry.core.init.BlockInit;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.util.RandomSource;
+import net.minecraft.world.item.context.UseOnContext;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.common.ToolAction;
+import net.minecraftforge.common.ToolActions;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class RubberTreeBlockStripped extends RubberTreeBaseBlock {
+    public RubberTreeBlockStripped(final Properties properties) {
+        super(properties);
+    }
+
+
+    @Override
+    public @Nullable BlockState getToolModifiedState(final BlockState state, final UseOnContext context, final ToolAction toolAction, final boolean simulate) {
+        if (toolAction == ToolActions.AXE_STRIP) {
+            return null;
+        }
+
+        return super.getToolModifiedState(state, context, toolAction, simulate);
+    }
+
+    @Override
+    public void randomTick(final @NotNull BlockState state, final @NotNull ServerLevel level, final @NotNull BlockPos pos, final @NotNull RandomSource source) {
+        super.randomTick(state, level, pos, source);
+        //cache the direction enum array
+        final Direction[] cachedDirections = Direction.values();
+        for (final Direction cachedDirection : cachedDirections) {
```

Comment:
This doesn't really do anything and imo just makes the code here a little more messy.
```suggestion
        for (final Direction cachedDirection : Direction.values()) {
``` 
Also renaming `cachedDirection` to just `direction` would be ideal.